cmd/artifact/command/add.go

```diff
@@ -87,7 +87,7 @@ func appendBuildSubCommand(options *Options) *cobra.Command {
 			if err != nil {
 				return nil
 			}
-			err = notifySlack(options, fmt.Sprintf(":white_check_mark: *Build* (%s:%s)", buildData.Image, buildData.Tag), slack.MsgColorYellow)
+			err = notifySlack(options, fmt.Sprintf(":white_check_mark: *Build* (<%s:image>)", fmt.Sprintf("%s:%s", buildData.Image, buildData.Tag)), slack.MsgColorYellow)
```

Comment:
Nested `Sprintf`s 👀 
Those are hard to read.

Can't we just do this instead?

```suggestion
			err = notifySlack(options, fmt.Sprintf(":white_check_mark: *Build* (<%s:%s:image>)", buildData.Image, buildData.Tag), slack.MsgColorYellow)
```